core/Symbols.cc

```diff
@@ -2732,6 +2735,11 @@ const ClassOrModule *ConstClassOrModuleData::operator->() const {
     return &symbol;
 };
 
+const ClassOrModule &ConstClassOrModuleData::operator*() const {
+    runDebugOnlyCheck();
+    return symbol;
+};
```

Comment:
Does having this method defeat the original purpose of the ClassOrModuleData wrapper classes?

I thought the idea was to detect whether the vector backing the ClassOrModule classes had been reallocated (e.g. in response to a new symbol being entered), and by having only the `->` methods, you would be forced to to the `runDebugOnlyCheck` on all field accesses to see whether something had changed, instead of only on the first dereference.

Should the implementation of `accumulateMembers` instead take a ClassOrModuleData, or just a `ClassOrModuleRef`?